test/rocjpeg_negative_api_tests/rocjpeg_api_negative_tests.cpp

```diff
@@ -334,7 +334,7 @@ int RocJpegApiNegativeTests::TestInvalidGetErrorName() {
     }
 
     // Scenario 2: Pass a valid error code and ensure it returns a non-null name
-    for (int i = 0; i >= -8; i--) {
+    for (int i = 0; i >= -12; i--) {
```

Comment:
Why are we using a hardcoded value here. Please add an enum (MAX_NUM_ERROR_CODES) in the RocJpegStatus and use that instead so we don't need to touch this code

Reply:
Done